src/marqo/tensor_search/models/settings_object.py

```diff
@@ -113,7 +114,14 @@
             "examples": [
                 5
             ]
-        }
+        },
+        NsFields.number_of_replicas: {
+            "type": "integer",
+            "minimum": 1,
```

Comment:
minimum number of replicas is actually 0 (which means just the index, without a replica)

Reply:
updated